packages/react-components/source/react/library/popover/Popover.js

```diff
@@ -12,7 +12,16 @@ const propTypes = {
   /** Component children */
   children: PropTypes.node,
   /** Side the arrow appears on */
-  side: PropTypes.oneOf(['top', 'bottom', 'left', 'right']),
+  side: PropTypes.oneOf([
```

Comment:
would it be worth going with `top-left`, `top-right` etc just to be more explicit?

Reply:
I had thought about it, but that would be a breaking change and would require a major version bump. We could target that kind of change in v6.